lib/operations.rb

```diff
@@ -9,7 +9,7 @@
 require "active_support/core_ext/module/delegation"
 require "active_support/inflector/inflections"
 require "active_model/naming"
-require "active_model/errors"
+require "active_model/errors" if RUBY_VERSION.to_d < 3.1
```

Comment:
Why did you add this? This doesn't solve the problem? I would keep 3.1 failing for a while

Reply:
Also, Gem::Version

Reply:
But but then how specs are green then?

Reply:
apparently ruby 3.1 does not need that :D

Reply:
but it can't be true :( the gem needs it

Reply:
Actually, maybe no indeed, can you drop the line completely?